store/index.js

```diff
@@ -16,11 +17,38 @@ export default class {
   @observable
   projects = []
 
+  @observable
+  months = [
+    {
+      id: 'fakeId414',
+      projectId: 'proj1',
+      startsAt: startOfMonth(new Date('2018-12-12T12:47:08.439Z')),
+      createdAt: new Date('2018-12-12T12:47:08.439Z'),
+      monthId: 11,
+    },
+    {
+      id: 'fakeId424d',
+      projectId: 'proj1',
+      startsAt: startOfMonth(new Date('2018-11-02T19:47:08.439Z')),
+      createdAt: new Date('2018-11-02T19:47:08.439Z'),
+      monthId: 10,
+    },
+    {
+      id: 'fakeIdbdx',
+      projectId: 'proj1',
+      startsAt: startOfMonth(new Date('2018-10-28T19:47:08.439Z')),
+      createdAt: new Date('2018-10-28T19:47:08.439Z'),
+      monthId: 9,
+    },
+  ]
+
   @observable
   selectedProject = 0
 
   @observable
-  selectedMonth = 0
+  selectedMonth = {
+    id: 10,
```

Comment:
since you're setting `selectedMonth` with: `props.store.setSelectedMonth(month.monthId)` it means it will have a value of monthId, which is string. Please change it so the default value for selectedMonth is also a string (better yet, it should be undefined by default), until it gets set.